README.md

```diff
@@ -226,6 +226,14 @@ await client.logon();
 await client.logoff();
 ```
 
+## Creating new instances
+Creates and returns a new instance of the specified entity
```

Comment:
I think this would warrant a more complete exemple, i.e. how to use such an instance.